metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj

```diff
@@ -423,39 +420,50 @@
             :errors ["Finding concept type [tag-association] with parameters [provider-id] is not supported."]}
            (util/find-concepts :tag-association {:provider-id "REG_PROV"})))))
 
-;; Failing in external Oracle
-;; CMR-4335 - services
-#_(deftest find-services
-    (let [serv1 (util/create-and-save-service 1 3)
-          serv2 (util/create-and-save-service 2 2)]
-      (testing "find latest revsions"
-        (are3 [servs params]
-              (= (set servs)
-                 (set (->> (util/find-latest-concepts :service params)
-                           :concepts
-                           (map #(dissoc % :provider-id :revision-date :transaction-id)))))
-              "with metadata search by concept-id"
-              [serv1] {:concept-id (:concept-id serv1)}
-
-              "with metadata search by native-id"
-              [serv2] {:native-id (:native-id serv2)}
-
-              "exclude metadata search by concept-id"
-              [(dissoc serv1 :metadata)] {:concept-id (:concept-id serv1)
-                                          :exclude-metadata true}
-
-              "exclude metadata search by native-id"
-              [(dissoc serv2 :metadata)] {:native-id (:native-id serv2)
-                                          :exclude-metadata true}
-
-              "no match"
-              [] {:native-id "foo"}))
-
-      (testing "find all revisions"
-        (let [num-of-servs (-> (util/find-concepts :service {})
-                              :concepts
-                              count)]
-          (is (= 5 num-of-servs))))))
+(deftest find-services
+  (let [;; create two services with the same native-id on different providers with multiple revisions
+        serv1 (util/create-and-save-service "REG_PROV" 1 3)
+        serv2 (util/create-and-save-service "SMAL_PROV1" 1 2)]
```

Comment:
just a nit, SMAL vs SMALL

Reply:
That was set up by the old code and the reason it is done this way is because the provider-id has a length limit of 10 characters.